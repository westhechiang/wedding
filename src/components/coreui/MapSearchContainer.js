import styled from 'styled-components';

const MapSearchContainer = styled.div`
  height: auto;
  width: ${props => (props.active ? '70%' : '50%')};
  display: flex;
  flex-direction: column;
  position: absolute;
  top: 10px;
  right: 10px;
  z-index: 5;
  transition: .2s ease-in-out width;
`;

export default MapSearchContainer;
