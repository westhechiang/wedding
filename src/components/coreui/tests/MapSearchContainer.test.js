import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MapSearchContainer from '../../../components/coreui/MapSearchContainer';

describe('<MapSearchContainer />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<MapSearchContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should match its "active" snapshot', () => {
    const wrapper = shallow(<MapSearchContainer active />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
