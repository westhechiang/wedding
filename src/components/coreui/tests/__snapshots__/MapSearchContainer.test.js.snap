// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapSearchContainer /> should match its "active" snapshot 1`] = `
<div
  className="sc-bdVaJa jtlHGY"
/>
`;

exports[`<MapSearchContainer /> should match its empty snapshot 1`] = `
<div
  className="sc-bdVaJa YUoyc"
/>
`;
